onnxruntime/core/framework/execution_frame.h

```diff
@@ -91,7 +91,7 @@ class IExecutionFrame {
   // Input and Output values are passed in by executors
   std::vector<OrtValue> all_values_;
 
-  const std::vector<int> fetch_mlvalue_idxs_;
+  const std::vector<int>& fetch_mlvalue_idxs_;
 };
```

Comment:
Was the copy of vector<int> a significant enough perf issue is warrant using a reference here? In the profiling I've done I've never noticed it (vs things like the cost of setting up the memory pattern planner in the ExecutionFrame ctor which could be reworked)